manim/mobject/types/image_mobject.py

```diff
@@ -49,6 +49,38 @@ def reset_points(self):
 
 
 class ImageMobject(AbstractImageMobject):
+    """Displays an Image from a numpy array or a file.
+
+    Examples
+    --------
+    .. manim:: ImageFromArray
+        :save_last_frame:
+
+        class ImageFromArray(Scene):
+            def construct(self):
+                image = ImageMobject(np.uint8([[0, 100, 30, 200],
+                                               [255, 0, 5, 33]]))
+                image.set_height(7)
+                self.add(image)
+
+    .. manim:: ImageFromFile
+          :save_last_frame:
+
+          class ImageFromFile(Scene):
+              def construct(self):
+                  # Use PIL when you want to import an image from the web
+                  import requests
+                  from PIL import Image
+                  img = Image.open(requests.get("https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png",
+                                                stream=True).raw)
+                  img_mobject = ImageMobject(img)
+                  # this line, when you want to import your Image on your machine
+                  # img_mobject = ImageMobject("<your image address>")
+                  img_mobject.scale(3)
+                  self.add(img_mobject)
```

Comment:
I don't like that we have this example relying on a working internet connection that requires `master` to have `cropped.png` -- it is what broke the documentation build for our first release this morning.

I am usually all for adding more examples, but I don't think that this is a good one. Would you be okay with simply removing it? As an (somewhat easier) alternative, you could add a `Parameters` section explaining that either an appropriate numpy array or the path to an image can be supplied when instantiating an `ImageMobject`.

Reply:
I agree, maybe using the file in the repo (in theory, the logo/ directory will not be torn apart from the documentations folder)

Reply:
This hasn't been changed yet, no?